backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label :stock_location_id %>
```

Comment:
Explicitly state a different model's attribute to avoid adding a translation for the attribute to the model?  This just seems kind of what the opposite of what I thought we were going for.